roles/include_components/tasks/main.yml

```diff
@@ -17,11 +17,15 @@
 
 - name: Track RPMs
   ansible.builtin.include_tasks: track_rpm.yml
-  loop: "{{ ic_rpms }}"
+  loop: "{{ include_components_rpms | default([]) }}"
   when:
     - item is not in
-      ( (ic_gits + ic_dev_gits) |
+      ( (include_components_gits + include_components_dev_gits) |
         map('regex_replace', '^(?:.*/(.+)|.*/([^/]+))$', '\\1') |
         map('regex_replace', 'python', 'python3') | list)
     - item in ansible_facts.packages
+
+- name: Track podman components
+  ansible.builtin.include_tasks: track_podman_component.yml
+  loop: "{{ include_components_podman | default([]) }}"
```

Comment:
I suggest to add a loop_var option to improve understanding of  track_podman_component.yml

Reply:
thanks @pierreblanc I like the idea and will add it

Reply:
done :)